Framework/Geometry/inc/MantidGeometry/Rendering/GeometryHandler.h

```diff
@@ -105,9 +105,12 @@ class MANTID_GEOMETRY_DLL GeometryHandler {
                         std::vector<uint32_t> &&faces);
   /// return the actual type and points of one of the "standard" objects,
   /// cuboid/cone/cyl/sphere
-  void GetObjectGeom(detail::ShapeInfo::GeometryShape &mytype,
-                     std::vector<Kernel::V3D> &vectors, double &myradius,
+  void GetObjectGeom(detail::ShapeInfo::GeometryShape &myType,
+                     std::vector<Kernel::V3D> &vectors, double &myRadius,
```

Comment:
I propose we drop the `my` part of the argument names (`type`, `radius` instead of `myType` or `myRadius`). We don't use such style in more 'modern' parts of Mantid anyhow.

Reply:
That's alright, I'll do that.